btree/btree.c

```diff
@@ -574,7 +574,10 @@
 #include "ut/ut.h"          /** struct m0_ut_suite */
 #endif
 
-#define AVOID_BE_SEGMENT    0
+#define AVOID_BE_SEGMENT		0
+#define M0_BTREE_TRICKLE_MULTIPLIER	2
+#define M0_BTREE_PURGE_MODE_ON		true
+#define M0_BTREE_PURGE_MODE_OFF		false
```

Comment:
Kindly use SPACES instead of TABS between the variable name and value.
Also values can be aligned a bit closer to the variable name

Reply:
Done.